hrm-domain/hrm-service/service-tests/contact/contactById.test.ts

```diff
@@ -15,10 +15,9 @@
  */
 
 import * as contactApi from '../../src/contact/contactService';
-import '../case-validation';
+import '../caseValidation';
```

Comment:
Are we moving to this new naming convention? I thought we were doing dash separated names in this repo :thinking:

Reply:
No - we agreed on camel case everywhere I think? https://github.com/techmatters/aselo-coding-standards/blob/main/TypeScript.md#42-module-names